packages/atomic/src/components/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -64,6 +70,9 @@ export class AtomicBreadbox implements InitializableComponent {
   @State() public error!: Error;
   @State() private isCollapsed = true;
 
+  @FocusTarget()
+  private breadcrumbRemovedFocus!: FocusTargetController;
```

Comment:
Based off the name, this property sounds like a boolean? Could either `focusedBreadcrumb` or `lastFocusedBreadcrumb` be clearer names?

Reply:
ah, I meant it like an event, like `onBreadcrumbRemoved`. I'll change it to `breadcrumbFocus`.